### PR TITLE
Downloads Manager: Added command "Reveal Latest Download in Finder"

### PR DESCRIPTION
**Pull request by infused-kim**
## Description

Added the command `Reveal Latest Download in Finder` to the `Downloads Manager` extension.

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/downloads-manager/src/reveal-latest-download.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+  if (downloads.length < 1) {
+    await showHUD("No downloads found");
+    return;
+  }
+
+  const latestDownload = downloads[0];
+
+  try {
+    revealFile(latestDownload.path);
```
Comment (thomaspaulmann) [inline extensions/downloads-manager/src/reveal-latest-download.tsx:15]: You should be able to use our built-in [showInFinder](https://developers.raycast.com/api-reference/utilities#showinfinder) function.

```suggestion
    await showInFinder(latestDownload.path);
```

**Review by thomaspaulmann — CHANGES_REQUESTED** [review]
Nice addition, thanks! I left a few suggestions. I was torn apart naming the command to `Show Latest Download` to match better our `Show in Finder` action but wasn't sure. Any thoughts? Alternatively, you could add a keyword "show" to the command to make sure that others would find it easily in the root search.

**Comment by infused-kim** [comment]
@thomaspaulmann thank you for the suggestions. I agree with everything and pushed a commit implementing them.

**Comment by thomaspaulmann** [comment]
Looks like you have an issue with code formatting. Could you run prettier locally again and fix the issue? Then we're ready to merge.

**Comment by infused-kim** [comment]
It should be good to go now. I have also updated the changelog.

**Comment by thomaspaulmann** [comment]
Thanks, let's ship it! Also, would be great if you can add yourself as `contributor` in the `package.json`. This way you see the extension on your Raycast profile ;-)

**Pull request merged**